src/assets/i18n/fr.json

```diff
@@ -40,11 +40,11 @@
   "Applications": "",
   "Applied Dataset Quota": "",
   "Apply Owner": "",
-  "Are you sure you want to delete \"{cert}\"?": "",
-  "Are you sure?": "",
-  "Attention": "",
+  "Are you sure you want to delete \"{cert}\"?": "Êtes-vous sûr de vouloir supprimer "{cert}\"?",
```

Comment:
Should be:
```
"Êtes-vous sûr de vouloir supprimer \"{cert}\"?"
```

Reply:
Oops my bad.
Do I have to change it myself ?

Reply:
Yes